start

```diff
@@ -237,25 +341,43 @@ haskell_toolchain_library(name = "base")
 # )
 
 # An example binary using the Prelude module from the
-# GHC base package, to print the hello world.
+# GHC base package, and zlib from stackage, to print the hello world.
 haskell_binary(
     name = "example",
     srcs = [":Example.hs"],
-    deps = [":base"],
+    deps = [":base", "@stackage//:zlib"],
+)
+
+# Load cabal rules
+load(
+    "@rules_haskell//haskell:cabal.bzl",
+    "haskell_cabal_binary",
+)
+
+# Example usage of alex, use 'bazel builds //:words' to execute it
+genrule(
+    name = "words",
+    srcs = ["@alex", "@alex_example//file"],
+    outs = ["Words.hs"],
+    cmd = "$(location @alex) $(location @alex_example//file) -o $(location Words.hs)",
 )
```

Comment:
Using Alex directly in a project, on the other hand, is much less common. So, I don't think there is much value in including such an example in the `start` script. I expect for most users it will just be a slight annoyance that they have to remove this code in their fresh project.

Reply:
I get your point, as a bazel newcomer I'd surely like having a bit more than a bit less in the `start` script, but that's not the targeted audience.

Reply:
Issue addressed in 8f723ad

Reply:
But it's also not @aherrmann's point. The `start` script is not meant to show you the 10% case. It's meant to get you started with the 80% case. Other places than this are more appropriate to document uncommon cases (such as using `alex` in a genrule), e.g. as an example in the examples directory.